src/openzaak/conf/includes/base.py

```diff
@@ -348,6 +348,19 @@
 #
 
 
+ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS = {
+    "use-creation-year": "openzaak.components.zaken.api.utils.generate_zaak_identificatie_with_creation_year",
+    "use-start-datum-year": "openzaak.components.zaken.api.utils.generate_zaak_identificatie_with_start_datum_year",
+}
+
+ZAAK_IDENTIFICATIE_GENERATOR = config(
+    "ZAAK_IDENTIFICATIE_GENERATOR",
+    default="use-start-datum-year",
+    help_text="Select one of the options defined in ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS: "
+    f"[{', '.join(ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS.keys())}] "
+    "that define how the ZAAK identificatie is generated.",
```

Comment:
To make the formatting of the values a bit clearer in the rendered docs
```suggestion
    help_text=(
        "The method of **Zaak.identificatie** generation. Possible values are: "
        f"{', '.join(f'``{opt}``' for opt in ZAAK_IDENTIFICATIE_GENERATOR_OPTIONS.keys())} "
    ),
```